build-ci/new_package.rst

```diff
@@ -21,8 +21,42 @@ then needs to be added to the :file:`ups/*.table` file (and possibly the
 :file:`ups/*.cfg` file) of one or more other packages in the stack where it is
 used.
 
+.. _lfs-repos:
+
+Handling ``git lfs`` backed repos
```

Comment:
In the [Git LFS documentation page](http://developer.lsst.io/en/latest/tools/git_lfs.html) we established a usage style  (that I think match's GitHub's) where we use 'Git LFS' (note capitalization and lack of monospace) to refer to the technology and only use `git lfs` to refer to the client itself.